### PR TITLE
Repository refactoring and ghub workflow

### PR DESCRIPTION
**Pull request by g-poveda**
this PR includes : 
- moving the source code to the folder airobas/ so that when pip install from source it is possible to import via ```from airobas.verif_pipeline import ...``` kind of source code. And also eases the imports in the tests/ folder for automatic test
- adapts the imports in the tutorials notebook
- add workflow testing the installation of dependencies and run pytest in tests/ folder (imports and pipeline call on rosenbrock function is tested)
- update of README with google colab links to tutorial.

### CODE REVIEW

**Review by agalametz2 — APPROVED** [review]

**Comment by agalametz2** [comment]
Accept changes

File: tests/test_rosenbrock.py
Comment (ryma-bmzz) [inline tests/test_rosenbrock.py:1]: not sure I get what is the difference between the tutorial and this test?

File: pyproject.toml
Comment (ryma-bmzz) [inline pyproject.toml:1]: since tensorflow is removed from the dependencies, we may need to specify in the readme to use tensorflow as backend for keras?

**Review by ryma-bmzz — APPROVED** [review]
All good from my side

File: tests/test_rosenbrock.py
Comment (g-poveda) [reply tests/test_rosenbrock.py:1]: mostly the same yes, but in the last commit i remove all the plots so it's smaller.
here we check if the pipeline and different subbrick don't run with exception, the most simple example i had was the one from the notebook

File: pyproject.toml
Comment (g-poveda) [reply pyproject.toml:1]: the version we use for keras (2.15) is the one that also install tensorflow as unique backend (tbc?) so it's ok for now, i think.

**Pull request merged**